include/dxc/DXIL/DxilResource.h

```diff
@@ -29,8 +29,19 @@ class DxilResource : public DxilResourceBase {
   static unsigned GetNumDimensionsForCalcLOD(Kind ResourceKind);
   /// Total number of offsets (in [-8,7]) necessary to access resource.
   static unsigned GetNumOffsets(Kind ResourceKind);
-  /// Whether the resource kind is texture.
+  /// Whether the resource kind is a texture. This does not include
+  /// FeedbackTexture.
```

Comment:
Can we just make this "FeedbackTextures"?  Same with `IsAnyTextureArray`, replacing "texture feedback arrays" with "FeedbackTextures".